strings/find.py

```diff
@@ -0,0 +1,25 @@
+# Find a string
+"""Task
+In this challenge, the user enters a string and a substring. You have to print the number of times that the substring
+occurs in the given string. String traversal will take place from left to right, not from right to left.
+
+NOTE: String letters are case-sensitive.
+"""
+
+
+def count_substring(string, sub_string):
+    count = 0
+    pos = string.find(sub_string)
+    for _ in range(0, len(string)):
+        while pos != -1:
+            pos = string.find(sub_string, pos + 1)
+            count += 1
+    return count
+
+
+if __name__ == '__main__':
+    string = input().strip()
+    sub_string = input().strip()
```

Comment:
I didn't find a requirement that strings should be stripped. 
What if I want to count the amount of whitespaces in the given string?

Reply:
strip() was used in a pattern of submission on hackerrank and I couldn't change it when I added my solution there